QC/wdl/tasks/deepvariant.wdl

```diff
@@ -0,0 +1,97 @@
+version 1.0
+
+workflow runDeepVariant {
+
+    call DeepVariant
+
+    output{
+        File deepVariantVCF    = DeepVariant.vcfOut
+        File deepVariantVCFTBI = DeepVariant.vcfIdxOut
+        File deepVariantHTML   = DeepVariant.visualReport
+    }
+}
+
+task DeepVariant{
+    input{
+        File inputReads
+        File inputReadsIdx
+        File assembly
+        File assemblyIndex
+        String sample
+
+        String modelType = "HYBRID_PACBIO_ILLUMINA"
+        String? callRegions
+
+        Int memSizeGB = 128
+        Int threadCount = 64
+        Int diskSizeGB = 128
+        String dockerImage = "google/deepvariant@sha256:440074e9cf854e20e3e05eab0a4fbbc32652c1f0d71b2aacbbd35da47f84faae" # 1.2.0
+
```

Comment:
Why so many newlines between the last argument and the }? The rest of the WDL doesn't do that.

(Yes, this is a nitpick... feel free to ignore!)

Reply:
cuz